Cargo.toml

```diff
@@ -16,7 +16,7 @@ ioctl-gen = "0.1.1"
 libc = "0.2.69"
 mmap = "0.1.1"
 rusttype = "0.8.2"
-evdev = "0.10.1"
+evdev = { git = "https://github.com/LinusCDE/evdev.git", rev = "43fb0b33994dba667ab3f1c107b7393ac3166ffe" }
```

Comment:
I also had to patch the evdev lib for now. As it wasn't able to detect the multitouch resolution.

Link to the issue is in this commit (https://github.com/canselcik/libremarkable/pull/48/commits/46a341899f6eceb4458746549d8f7be23ec8d869).